Packs/EWS/ReleaseNotes/1_8_27.md

```diff
@@ -0,0 +1,5 @@
+
+#### Integrations
+##### EWS v2
+- Fixed an issue where ***fetch-incidents*** sometimes failed with an `UnknownTimeZoneError`.
+- Updated the Docker image to: *demisto/py-ews:1.0.0.22402*.
```

Comment:
```suggestion
- Updated the Docker image to: *demisto/py-ews:1.0.0.23126*.
```

Reply:
Good catch

Reply:
pushed a commit and also updated the fix line with the version that the issue was introduced in.